ios/App/AudiobookshelfUnitTests/Shared/player/util/PlayerTimeUtilsTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PlayerTimeUtilsTests.swift
+//  AudiobookshelfUnitTests
+//
+//  Created by Ron Heft on 9/20/22.
+//
+
+import XCTest
+@testable import Audiobookshelf
+
+final class PlayerTimeUtilsTests: XCTestCase {
```

Comment:
These are the tests, using all the testing utils available in Swift. I expect I will eventually add a mocking framework, but going to try and keep things simple for now.